Documentation/supported-mod-list.md

```diff
@@ -18,324 +21,328 @@
-13. [Kleiners Adventures Demo](https://www.moddb.com/mods/kleiners-adventures)
-14. [Kleiners Adventures: The White Line Demo](https://www.moddb.com/mods/kleiners-adventure-siemka321-edition) 
-15. [LV-426: Episode 1](https://www.runthinkshootlive.com/posts/lv426-version-a/)
-16. [Malevolence(1.3 and older)](https://www.moddb.com/mods/malevolence)
-17. [Meth-Life Demo v1.0](https://www.gamewatcher.com/mods/half-life-mod/meth-life-mod-demo-1-0)
-18. [Night-Fire Demo](https://efreeman1.narod.ru/Index.htm)
-19. [On the Other Side](https://twhl.info/competitions.php?results=14) aka *OTOS* (there is a train-related bug in the beginning, which makes you to use **noclip**, but it's an inner flaw of the map)
-20. [Orion: Part 1( Project Orion )](https://www.moddb.com/mods/orionold)
-21. [Padle Mesto Demo](https://www.moddb.com/games/half-life/addons/padle-mesto-demo) (some scripts & buttons are not configured properly in this mod, so they can be activated/deactivated automaically after reloading of a saved game)
-22. [Panic at Black Mesa Demo](https://www.moddb.com/mods/half-life-panic-at-black-mesa)
-23. [Plan B v1.1](https://www.fileplanet.com/129198/120000/fileinfo/Plan-B-SP-Mappack)
-24. [Project Focus North v1.6 Demo](https://www.fileplanet.com/84637/80000/fileinfo/Project-Focus-North)
-25. [Qortez( Quortez )](https://www.runthinkshootlive.com/posts/qortez/) (it's recommended to use only default low-poly model of scientist for this mod; also set **fps_max** to **60** via console to prevent an issue with one of scripted sequences)
-26. [Return to Lambdacore Demo](https://www.moddb.com/games/half-life/addons/return-to-lamdacore)
-27. [Route City Beta 1](https://www.moddb.com/mods/route-city)
-28. [SHAFT - Part 1](https://hl.loess.ru/?mod=913)
-29. [Shortcut v1.0 Beta](https://www.runthinkshootlive.com/posts/shortcut/)
-30. [Stoka](https://www.ceskemody.cz/mapy.php?lng=1&clanek=222&razeni_hra=1&pn=stoka)
-31. [Striker's Compo 26](https://twhl.info/vault.php?map=5190) (buggy)
-32. [Technology Test 2](https://web.archive.org/web/20110512012317/http://www.richmans-maps.ch.vu/)
-33. [The Gate Playable Demo](https://www.fileplanet.com/116347/110000/fileinfo/The-Gate-Playable-Demo)
-34. [They are Back](https://www.runthinkshootlive.com/posts/they-are-back/)
-35. [Tiefseelabor( Deep Sea Laboratory )](https://www.runthinkshootlive.com/posts/deep-sea-laboratory/)
-36. [Time-Shift](https://web.archive.org/web/20110512012317/http://www.richmans-maps.ch.vu/)
-37. [Train Single Beta](https://csm.dev/threads/half-strike-rezultaty.36394/) (Remove **gfx.wad** from **TrainSingle** folder)
-38. [WAR: The Killer Beta 0.1](https://www.moddb.com/mods/war)
-39. [White Force Beta( Residual Point prototype )](https://www.moddb.com/mods/hl-residual-point/downloads/white-force-beta-2002)
-40. [Your First Mission Demo](https://twhl.info/forums.php?thread=3925)
-41. [Zombieland v1.1](https://www.moddb.com/mods/zombieland)
+4. Black Meat (unfinished mappack of 3 maps)
```

Comment:
There is no link to it anymore?

Reply:
Probably, It's exist but all links dead.
@Qwertyus3D has some mod names without links in his list.